build/yamls/antrea.yml

```diff
@@ -5692,7 +5692,7 @@ data:
     #                    the PSK value must be passed to Antrea Agent through an environment
     #                    variable: ANTREA_IPSEC_PSK.
     # - wireGuard:       Enable WireGuard for tunnel traffic encryption.
-    trafficEncryptionMode: "none"
+    trafficEncryptionMode: "wireGuard"
```

Comment:
Is it for testing only?

Reply:
oops I committed all local changes when I addressed your comment above
should be reverted now